inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/querybuilder/SPARQLQueryBuilder.java

```diff
@@ -663,19 +663,19 @@ public SPARQLQueryBuilder withLabelMatchingExactlyAnyOf(String... aValues)
         IRI ftsMode = forceDisableFTS ? FTS_NONE : kb.getFullTextSearchIri();
         
         if (FTS_LUCENE.equals(ftsMode)) {
-            addPattern(PRIMARY, withLabelMatchingExactlyAnyOf_RDF4J_FTS(aValues));
```

Comment:
Hm, all these changes where the underscore is removed, I'm not sure if this is actually an improvement in readability. The underscore somehow chunked the long variable names so they are easier to read, in my opinion.